.bazelrc

```diff
@@ -1,2 +1,2 @@
-build --cxxopt=-std=c++14
+build --cxxopt=-std=c++14 --cxxopt=-O3 --cxxopt=-Werror --cxxopt=-Wall
```

Comment:
Opened an issue up about cleaning up some of these build flags! May be good to get someone familiar with the build system/infrastructure.